src/doom/am_map.c

```diff
@@ -295,6 +295,16 @@ cheatseq_t cheat_amap = CHEAT("iddt", 0);
 
 static boolean stopped = true;
 
+// [crispy] Antialiased lines from Heretic with more colors
```

Comment:
If possible, please turn all the functions that you add or modify into `static` functions.